Example/Tests/GINISupportedFormatsViewControllerTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  GINIsupportedFormatsViewControllerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 10/19/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class GINISupportedFormatsViewControllerTests: XCTestCase {
+    
+    let supportedFormatsViewController = SupportedFormatsViewController(style: .plain)
+    
+    override func setUp() {
+        super.setUp()
+        _ = supportedFormatsViewController.view
+    }
+    
+    func testSectionsCount() {
+        let sectionsCount = supportedFormatsViewController.sections.count
```

Comment:
I don't think this test is enough. It should probably be here as well, but not as the only one.
It just tests that the `numberOfSections` returns `sections.count`. I'd like to see something more concrete... like "Does it contain the 5 sections I expect". I think it's fine to have hardcoded values in the tests. You might want to avoid them if you can, but here, I'd prefer to explicitly test against a concrete expectation about the table.

If you add another section, let the test fail. It would make it explicit that you're changing the class' behaviour.